tests/components/mqtt/test_init.py

```diff
@@ -1254,7 +1253,6 @@ async def test_mqtt_ws_get_device_debug_info_binary(
         "unique_id": "unique",
     }
     data = json.dumps(config)
-    config["platform"] = mqtt.DOMAIN
```

Comment:
The `platform` is is no longer added to the discovery message, see:
https://github.com/home-assistant/core/pull/118757/files#diff-189ecd07b98c2a9602a647f97dfe63e919c95af2d3856701044958b230591ef1L274